pkg/util/k8s/k8s.go

```diff
@@ -1836,3 +1836,173 @@ func GetV1beta1CRDFromFile(
 	}
 	return crd, nil
 }
+
+// GetAllObjects gets all objects from k8s
+func GetAllObjects(k8sClient client.Client, namespace string) ([]client.Object, error) {
```

Comment:
nit: Do we really need to go overboard and include all the objects?
I guess just secrets, configmaps, services, endpoints and storageclasses should do it.